celeri.py

```diff
@@ -2290,17 +2290,41 @@ def test_okada_equals_cutde():
             obs_pts=obs_coords, tris=tri_coords, nu=poissons_ratio
         )
     slip = np.array([[strike_slip, dip_slip, tensile_slip]])
-    disp = disp_mat.reshape((-1, 3)).dot(
-        slip.flatten()
-    )  # TODO: #42 I think this and the line below are wrong???
-    u_cutde = np.sum(disp.reshape(-1, 3), axis=1)
-    # u_x_cutde[i] = 0
-    # u_y_cutde[i] = 0
-    # u_z_cutde[i] = 0
-
-    # TODO: Try grid of observation points
-    # TODO: Try positive z thing again
-    # TODO: Check sign of slip
-    print(disp_mat.shape)
-    print(disp.shape)
-    print(u_cutde.shape)
+
+    # What to do with array dimensions here? We want to compute the
```

Comment:
Hopefully this helps explain the array dimensions and how you might debug this in the future!